skyfield/timelib.py

```diff
@@ -210,19 +210,21 @@ class Time(object):
     eps_correction = 0.0
 
     def __init__(self, ts, tt):
-        self.tt = tt
+        self.tt = _to_array(tt)
         self.ts = ts
         self.shape = getattr(tt, 'shape', ())
 
     def __len__(self):
         return self.shape[0]
 
     def __repr__(self):
-
-        if getattr(self.tt, 'shape', ()):
+        if self.tt.size == 0:
+            return '<Time 0 values>'
```

Comment:
Nice, I'm glad that this will no longer cause an error — thanks! I'm still sometimes tweaking the repr's and str's, so I might adjust this more later, but for now it should do fine.